### DISCUSSION TITLE
服务配置和状态的获取

### DISCUSSION THREAD

**Discussion by hexergogo** [bfenetworks/bfe#1043] [Q&A (问题和回答)] [answered]
有什么方法可以让我用 cluster name 获取到配置中真实的ip:port， 而不是通过数据库查询， 还有获取当前服务的状态(all backed is down) , 我想要做个接口用于查看检查服务的信息和状态

**Comment by ohscartao** [comment, marked as answer]
1，后端实例信息
    推荐使用https://github.com/bfenetworks/api-server，这是控制面的API服务。
    每个cluster会关联到sub-cluster，每个sub-cluster关联到一个实例池（pool）。API server可以提供对应接口，查询相关的所有信息。

2，服务后端的状态，目前并没有给出直接支持
    这个问题的困难之处在于，一个BFE集群中，多个BFE对后端实例的状态判定可能是不一致的。

**Reply by hexergogo** [reply]
感谢
